packages/model-fragments/lib/fragments/ext.js

```diff
@@ -172,7 +172,6 @@ Model.reopen({
       if (this._fragments[key]) {
         this._fragments[key].rollback();
       }
-      this._fragments[key] = null;
```

Comment:
@igorT I'm not sure anymore [why this was added originally](https://github.com/lytics/ember-data.model-fragments/pull/34/files#diff-b8af553d86227a5fda84f3e123c722e3R972). It doesn't seem to affect any other tests, would you mind [testing it in your app](https://rawgit.com/lytics/ember-data.model-fragments/multiple-rollback-tmp-dist/dist/ember-data.model-fragments.js)?